spec/factories/organisation.rb

```diff
@@ -11,17 +11,15 @@
     holds_own_stock { true }
   end
 
-  factory :organisation_la do
-    organisation
-    ons_code { "E07000041" }
-    created_at { Time.zone.now }
-    updated_at { Time.zone.now }
-  end
-
   factory :organisation_rent_period do
     organisation
     rent_period { 2 }
     created_at { Time.zone.now }
     updated_at { Time.zone.now }
   end
+
+  factory :organisation_relationship do
+    child_organisation { FactoryBot.create(:organisation) }
+    parent_organisation { FactoryBot.create(:organisation) }
+  end
```

Comment:
put this back in the organisation file, I think that's simpler than having a separate organisation_relationship file especially as organisation_rent_period is here too